rust/protocol/src/sealed_sender.rs

```diff
@@ -1117,6 +1482,15 @@ pub async fn sealed_sender_decrypt_to_usmc(
             authentication_tag,
             encrypted_message,
         } => {
+            let encrypted_message_key: [u8; 32] = encrypted_message_key
+                .as_ref()
```

Comment:
(unneeded again)

Reply:
Hmmm, I'm getting this when I try to remove it:
```rust
error[E0277]: the trait bound `[u8; 32]: From<Box<[u8]>>` is not satisfied
    --> rust/protocol/src/sealed_sender.rs:1483:39
     |
1483 |                 encrypted_message_key.try_into().map_err(|_| {
     |                                       ^^^^^^^^ the trait `From<Box<[u8]>>` is not implemented for `[u8; 32]`
```

Reply:
Oh, hm, fair. Okay, `as_ref()` is reasonable here.